### PR TITLE
updated build matrix

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

**Comment by lsmith77** [comment]
looks like we have some work to do in the composer.json to fix the lower deps bound:
https://travis-ci.org/liip/LiipThemeBundle/jobs/87228218

**Comment by lsmith77** [comment]
@thormeier / @dbu can one of you review this and merge?

File: .travis.yml [hunk 1]
```diff
@@ -5,31 +5,40 @@ php:
...
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache
```
Comment (stof) [inline .travis.yml]: you should persist only the ``files`` subfolder of the cache

File: .travis.yml [hunk 2]
```diff
@@ -5,31 +5,40 @@ php:
...
   include:
-    - php: 5.5
+    - php: 5.6
       env: SYMFONY_VERSION=2.3.*
-    - php: 5.5
+    - php: 5.6
       env: SYMFONY_VERSION=2.4.*
-    - php: 5.5
+    - php: 5.6
       env: SYMFONY_VERSION=2.5.*
```
Comment (stof) [inline .travis.yml]: you should remove testing against 2.4, 2.5 and 2.6 which are EOLed IMO, to limit the number of jobs in the matrix

File: .travis.yml [hunk 3]
```diff
@@ -5,31 +5,40 @@ php:
...
-    - php: 5.5
-      env: SYMFONY_VERSION=2.7.*
-    - php: 5.5
-      env: SYMFONY_VERSION='2.8.*@dev'
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION=3.0.*@dev
+    - php: 5.3
+      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
```
Comment (stof) [inline .travis.yml]: changing your Symfony constraint while testing lowest deps does not make sense, as you also change the lowest bound

Comment (lsmith77) [inline .travis.yml]: ie. I should remove ``SYMFONY_VERSION=2.3.*``?

File: .travis.yml (hunk 2 above)
Comment (lsmith77) [inline .travis.yml]: yeah .. might make sense indeed.

File: .travis.yml (hunk 1 above)
Comment (lsmith77) [inline .travis.yml]: ie. ``- $HOME/.composer/cache/files`` ?

**Comment by dbu** [comment]
:+1:

**Pull request merged**